cves/kernel/CVE-2013-1827.yml

```diff
@@ -464,11 +444,14 @@ CWE_instructions: |
     CWE: ["123", "456"] # this is ok
     CWE: [123, 456]     # also ok
     CWE: 123            # also ok
-CWE:
-CWE_note:
+CWE: 476
```

Comment:
It was mentioned previously a few times, but if this has a possibility of privilege elevation, should that be mentioned here as well as a CWE?